c2corg_ui/static/js/addassociationdirective.js

```diff
@@ -55,7 +56,6 @@ app.AddAssociationController = function(appApi, $attrs) {
   this.api_ = appApi;
 
 
-  this.dataset = $attrs['dataset'];
```

Comment:
I noticed that I don't have to - the only place I use it is in the directive itself, where I can get it directly from attrs.dataset, so finally there is no use for this.dataset in the controller.